tests/EdFi.Ods.WebApi.IntegrationTests/UriHelper.cs

```diff
@@ -0,0 +1,93 @@
+﻿// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+namespace EdFi.Ods.WebApi.IntegrationTests
+{
+    internal class UriHelper
```

Comment:
We have this duplicated now between the two projects, should the url constants move into Test.Common?

Reply:
Yes, i will make it a class we can new and pass in the base address